.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Link
+<!-- A link to the shard's source -->
+
+## Description
```

Comment:
Let's remove the description. Anyway, the reviewer will need to follow the link the read readme/understand the code.